onnxruntime/python/onnxruntime_collect_build_info.py

```diff
@@ -0,0 +1,99 @@
+# -------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+# --------------------------------------------------------------------------
+import warnings
+import ctypes
+import sys
+
+
+def find_cudart_versions(build_env=False):
+    # ctypes.CDLL and ctypes.util.find_library load the latest installed library.
+    # it may not the the library that would be loaded by onnxruntime.
+    # for example, in an environment with Cuda 11.1 and subsequently
+    # conda cudatoolkit 10.2.89 installed. ctypes will find cudart 10.2. however,
+    # onnxruntime built with Cuda 11.1 will find and load cudart for Cuda 11.1.
+    # for the above reason, we need find all versions in the environment and
+    # only give warnings if the expected cuda version is not found.
+    # in onnxruntime build environment, we expected only one Cuda version.
+    if 'linux' not in sys.platform:
+        warnings.warn('find_cudart_versions only works on Linux')
+        return None
```

Comment:
Please use the same function from util:
[https://github.com/microsoft/onnxruntime/blob/8d737f977056444a307f1b7f0bcd402fba62d790/tools/python/util/__init__.py](https://github.com/microsoft/onnxruntime/blob/8d737f977056444a307f1b7f0bcd402fba62d790/tools/python/util/__init__.py)

Reply:
I cannot use it because find_cudart_versions is not just called at package time. It is also called at runtime when is_linux is not available.